packages/dso-toolkit/src/components/form/form-group.scss

```diff
@@ -79,34 +79,34 @@
     // Reposition feedback icon if input has visible label above
     label {
       & ~ .form-control-feedback {
-        top: (typography.$line-height-computed + 5); // Height of the `label` and its margin
+        inset-block-start: (typography.$line-height-computed + 5); // Height of the `label` and its margin
```

Comment:
Is er ook een logical property voor line-height? Of hoe werkt dat eigenlijk? Is het logisch dat we een logical property koppelen aan een line-height variable?

Reply:
Ok prima!

Ik weet niet of ik het als misbruik wil bestempelen: Kennelijk zit er een relatie tussen de positionering en de line-height. Nu we overstappen naar logical properties, wat gebeurt er als de logische richtingen wijzigen?

Of klappen teksten dan ook om, waardoor line-height nog steeds klopt?

Reply:
alles klapt om

Reply:
en die waarde van `inset-block-start` die uit de variabele komt is hard 29px, dus die gaat nooit met de line-height en/of font-size mee, dus die relatie is er eigenlijk niet.

Reply:
Check, helder.